### PR TITLE
Added target module

### PR DESCRIPTION
**Pull request by DarKnight24**

### CODE REVIEW

**Comment by viyatb** [comment]
Add a file to install the fuzzy lib - otherwise build fails. :)

**Comment by DarKnight24** [comment]
I have added a install function. Use it to install ssdeep. Then have look at the target module :).

**Comment by viyatb** [comment]
Also are you using a 4 space indentation OR 8?

File: install.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+)
+
+func install() {
+	if os.Getegid() != 0 {
+		fmt.Println("Please run the program with sudo privileges")
+		os.Exit(1)
+	}
+	fmt.Println("Installing ssdeep package...")
+	command := exec.Command("/bin/sh", "-c", "apt-get install ssdeep")
+	if err := command.Run(); err != nil {
```
Comment (viyatb) [inline install.go:28]: Modify this to install `ssdeep` using the source - this is so it works in all *nix distributions.

**Comment by DarKnight24** [comment]
I am using 4 space indentation.

File: install.go [hunk 2]
```diff
@@ -4,20 +4,40 @@ import (
...
+	complete := true
+	fmt.Println("[-]Installing ssdeep package...")
+	command := exec.Command("/bin/sh", "-c", `wget http://downloads.sourceforge.net/project/ssdeep/ssdeep-2.13/ssdeep-2.13.tar.gz; \
+		tar zxvf ssdeep-2.13.tar.gz; cd ssdeep-2.13; ./configure; make; make install `)
+	command.Stdout = os.Stdout
+	command.Dir = tempPath
 	if err := command.Run(); err != nil {
-		fmt.Println(err)
+		utils.Perror("Unable to install ssdeep")
+		complete = false
```
Comment (viyatb) [inline install.go:31]: Add a `Perror` command here to print `wget http://downloads.sourceforge.net/project/ssdeep/ssdeep-2.13/ssdeep-2.13.tar.gz; tar zxvf ssdeep-2.13.tar.gz; cd ssdeep-2.13; ./configure; make; make install` on the console for user to run it manually if it does not run from the app itself.

**Pull request merged**